sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/implementation/UtilsImpl.java

```diff
@@ -85,12 +83,18 @@ public final class UtilsImpl {
 
     public static final HttpHeaderName DOCKER_DIGEST_HEADER_NAME = HttpHeaderName.fromString("docker-content-digest");
     // TODO (limolkova) should we send index and list too so that we won't need to change the default later on?
-    public static final String SUPPORTED_MANIFEST_TYPES = ManifestMediaType.OCI_MANIFEST + "," + ManifestMediaType.DOCKER_MANIFEST;
+    public static final String SUPPORTED_MANIFEST_TYPES = "*/*"
```

Comment:
Question: does the order matter here?  I have a different order for .NET, so if it's important that they are sent in this order, I would want to change what we have.

Reply:
Bin mentioned order is not important.